### PR TITLE
Remove label element binding

### PR DESCRIPTION
**Pull request by tomalec**
https://github.com/Starcounter/uniform.css/issues/99
https://github.com/Starcounter/uniform.css/issues/74

### CODE REVIEW

File: components/uni-date-picker/index.html [hunk 1]
```diff
@@ -78,19 +78,26 @@ <h3>Plain date picker</h3>
         <vaadin-date-picker></vaadin-date-picker>
     </div></div>
 
-    <h3>Label attribute</h3>
+    <h3>Label <s>attribute</s> element</h3>
+    <p>
+        We consider label as attribute an anti-patern,
+        and suggest the use of native <code>&lt;label&gt;</code> element.
+    </p>
```
Comment (warpech) [inline components/uni-date-picker/index.html]: I think there is no need for negativity about a Vaadin feature that we don't use.

I propose to change it to something like below:

> ## Labelling of the control
>
> A native <code>&lt;label&gt;</code> element should be used to provide a label to the <code>&lt;input type="date"&gt;</code>.
> 
> Note that Vaadin date picker includes an implementation of a label within `vaadin-date-picker`, but the Uniform component does not use that.

File: components/uni-date-picker/material.html [hunk 2]
```diff
@@ -86,22 +86,26 @@ <h3>Plain date picker</h3>
...
     </div></div>
 
-    <h3>Label attribute</h3>
-    <p>You can use regular CSS rules to style, like <code>uni-date-picker {width: 256px; height: 100px;}</code></p>
-
+    <h3>Label <s>attribute</s> element</h3>
+    <p>
+        We consider label as attribute an anti-patern,
+        and suggest the use of native <code>&lt;label&gt;</code> element.
+    </p>
```
Comment (warpech) [inline components/uni-date-picker/material.html]: See my previous comment.

File: components/uni-date-picker/index.html (hunk 1 above)
Comment (tomalec) [reply components/uni-date-picker/index.html]: I was just trying to follow Vaadin element's demo structure and titles

**Comment by tomalec** [comment]
Please also note https://github.com/Starcounter/uniform.css/issues/99#issuecomment-392700581

**Pull request merged**